packages/insight/src/components/pill.tsx

```diff
@@ -1,23 +1,30 @@
 import { FC } from 'react';
-import styled from 'styled-components';
+import styled, { useTheme } from 'styled-components';
 import CloseLightSvg from 'src/assets/images/close-light.svg'
-import {Black, Slate30} from '../assets/styles/colors';
+import {Black, Slate30, White} from '../assets/styles/colors';
+import { isMobile } from 'src/utilities/helper-methods';
 
 const PillBubble = styled.div`
-  padding: 7px;
-  margin-right: 10px;
   display: flex;
   align-items: center;
-  height: 40px;
+  ${isMobile() ? 
```

Comment:
Please do something like this instead:
```
@media screen and (max-width: ${size.size.mobileL}) {}
```
It preserves consistency with how the rest of the project handles mobile styles and utilizes native css media queries.

Check this file for reference: `src/assets/styles/transaction.ts`